### DISCUSSION TITLE
Hover effect jittering

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#8061] [Help] [answered]
I use `hover:` class to create a hover effect but it didn't work as I expected: https://play.tailwindcss.com/IUf7vgFJBV
besides the jittering, another problem I have is the effect is following my mouse position, the desired result will be when the darker rectangle is hovered, it shrinks all the way to the left until it becomes invisible

**Comment by alexanderbluhm** [comment, marked as answer]
Hi @Chandler-Zhu 
This is a very common problem. As a rule of thumb you can memorize that you don't want to add _hover effects directly_ to elements you _transform_.
A common example is to move an icon a bit up if you hover it.

Instead you can use `group` and `group-hover:` to transform a child element when the parent is hovered.
https://play.tailwindcss.com/gECoPtW7YA
